client/dashboard/app-dotcom/index.tsx

```diff
@@ -14,53 +15,55 @@ import type {
 } from '@automattic/api-core';
 import './style.scss';
 
-boot( {
-	name: 'WordPress.com',
-	basePath: '/v2',
-	mainRoute: '/sites',
-	Logo,
-	supports: {
-		sites: {
-			deployments: true,
-			performance: true,
-			monitoring: true,
-			logs: true,
-			backups: true,
-			scan: true,
+export default function () {
+	boot( {
+		name: 'WordPress.com',
+		basePath: isEnabled( 'dashboard/v2' ) ? '/v2' : '/',
```

Comment:
Two questions:

- It seems old v1 links like `/sites` etc do not work in the Calypso live URL. Is this expected? I guess this is a blocker.
- I think we need to also fix `isDashboardBackport()`?

https://github.com/Automattic/wp-calypso/blob/a0c2bbd4893fe2ca5fe19996eed41ba4f75c6137/client/dashboard/utils/is-dashboard-backport.ts#L1-L3